src/main/res/values-pa/strings.xml

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="title_apps">ਐਪਸ</string>
+    <string name="title_settings">ਸੈਟਿੰਗਜ਼</string>
+    <string name="title_shares">ਸ਼ੇਅਰ</string>
+    <string name="title_no_server">ਕੋਈ ਸਰਵੇ ਨਹੀਂ</string>
+    <string name="title_upload_list">ਅਪਲੋਡ ਕਰੋ</string>
+    <string name="title_offline_files">ਆਫਲਾਈਨ ਫਾਇਲਾਂ</string>
+    <string name="title_recent_files">ਹਾਲੀਆ ਫਾਈਲਾਂ</string>
+
+    <string name="button_search_application">ਗੂਗਲ ਪਲੇ ਖੋਜੋ</string>
+    <string name="button_sign_in">ਸਾਈਨ - ਇਨ</string>
+    <string name="button_yes">ਹਾਂ</string>
+    <string name="button_no">ਨਹੀਂ</string>
+    <string name="button_retry">ਦੁਬਾਰਾ ਕੋਸ਼ਿਸ਼ ਕਰੋ</string>
+
+    <string name="hint_password">ਪਾਸਵਰਡ</string>
+    <string name="hint_username">ਉਪਯੋਗਕਰਤਾ ਨਾਮ</string>
+
+    <string name="menu_back">ਵਾਪਸ</string>
+    <string name="menu_forward">ਅੱਗੇ</string>
+    <string name="menu_navigation_close">ਨੈਵੀਗੇਸ਼ਨ ਬੰਦ ਕਰੋ</string>
+    <string name="menu_navigation_open">ਖੁੱਲਾ ਨੇਵੀਗੇਸ਼ਨ</string>
+    <string name="menu_settings">ਸੈਟਿੰਗਜ਼</string>
+    <string name="menu_share">ਸਾਂਝਾ ਕਰੋ</string>
+    <string name="menu_save">ਡਿਵਾਈਸ ਤੇ ਸੇਵ ਕਰੋ</string>
+    <string name="menu_delete">ਮਿਟਾਓ</string>
+    <string name="menu_offline">ਆਫਲਾਈਨ </string>
+    <string name="menu_sort">ਲੜੀਬੱਧ</string>
+    <string name="menu_search">ਖੋਜ</string>
+    <string name="menu_open">ਖੁੱਲਾ</string>
+    <string name="menu_audio_files">ਫਾਇਲਾਂ</string>
+    <string name="menu_video_subtitle">ਉਪਸਿਰਲੇਖ</string>
+
+    <string name="message_empty_apps">ਕੋਈ ਐਪਸ ਨਹੀਂ</string>
+    <string name="message_empty_files">ਕੋਈ ਫਾਈਲਾਂ ਨਹੀਂ</string>
+    <string name="message_empty_shares">ਕੋਈ ਸ਼ੇਅਰ ਨਹੀਂ</string>
+    <string name="message_empty_recent_files">ਕੋਈ ਹਾਲੀਆ ਫਾਈਲਾਂ ਨਹੀਂ</string>
+
+    <string name="message_error_account_exists">ਖਾਤਾ ਪਹਿਲਾਂ ਹੀ ਮੌਜੂਦ ਹੈ</string>
+    <string name="message_error_search_application">ਸਾਨੂੰ ਇਸ ਫਾਈਲ ਲਈ ਕੋਈ ਸਮਰਥਿਤ ਐਪਸ ਨਹੀਂ ਮਿਲੇ ਹਨ</string>
+    <string name="forgot_password"> <a href="https://www.amahi.org/forgot">ਆਪਣਾ ਪਾਸਵਰਡ ਭੁੱਲ ਗਏ?</a></string>
+    <string name="message_error_authentication">ਉਪਯੋਗਕਰਤਾ ਨਾਮ ਜਾਂ ਪਾਸਵਰਡ ਸਹੀ ਨਹੀਂ ਹੈ. ਮੁੜ ਕੋਸ਼ਿਸ ਕਰੋ ਜੀ.</string>
+    <string name="message_error_authentication_connection">ਕੁਨੈਕਸ਼ਨ ਗਲਤੀ. ਆਪਣੀ ਸੈਟਿੰਗ ਦੀ ਜਾਂਚ ਕਰੋ ਅਤੇ ਦੁਬਾਰਾ ਕੋਸ਼ਿਸ਼ ਕਰੋ.</string>
+    <string name="message_error_authentication_field_empty">ਉਪਯੋਗਕਰਤਾ ਨਾਮ ਜਾਂ ਪਾਸਵਰਡ ਖੇਤਰ ਖਾਲੀ ਨਹੀਂ ਛੱਡਿਆ ਜਾ ਸਕਦਾ</string>
+    <string name="message_error_connection">ਕੁਨੈਕਸ਼ਨ ਗਲਤੀ</string>
+    <string name="message_tap_to_retry">ਦੁਬਾਰਾ ਕੋਸ਼ਿਸ਼ ਕਰਨ ਲਈ ਟੈਪ ਕਰੋ</string>
+    <string name="message_error_video">ਵੀਡੀਓ ਚਲਾਉਣ ਦੌਰਾਨ ਗਲਤੀ</string>
+    <string name="message_error_amahi_anywhere_app">Amahi Anywhere ਤੁਹਾਡੇ ਸਰਵਰ ਵਿੱਚ ਐਪ ਪਹੁੰਚਯੋਗ ਨਹੀਂ ਹੈ. ਕਿਰਪਾ ਕਰਕੇ ਇਹ ਸੁਨਿਸ਼ਚਿਤ ਕਰੋ ਕਿ ਤੁਹਾਡੇ ਨੈਟਵਰਕ ਵਿੱਚ Amahi Anywhere ਐਪ ਸਥਾਪਿਤ, ਚੱਲ ਰਹੀ ਅਤੇ ਪਹੁੰਚਯੋਗ ਹੈ.</string>
```

Comment:
English inside a string. We are translating Amahi at other places too. Please correct this.